src/SFA.DAS.ApprenticeFeedback.Api/Controllers/ApprenticeFeedbackTargetController.cs

```diff
@@ -68,5 +69,20 @@ public async Task<IActionResult> ProcessUpdate([FromBody] UpdateApprenticeFeedba
                 return BadRequest();
             }
         }
+
+        [HttpGet("RequiresUpdate")]
+        public async Task<IActionResult> GetFeedbackTargetsForUpdate([FromQuery] GetApprenticeFeedbackTargetsForUpdateQuery request)
+        {
+            try
+            {
+                var result = await _mediator.Send(request);
+                return Ok(result.ApprenticeFeedbackTargets);
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, $"Error attempting to retrieve apprentice feedback targets for update.");
+                throw;
```

Comment:
Should we return bad request / internal server error or just let the pipeline break?